tests/test_numpy_dtypes.py

```diff
@@ -169,6 +169,9 @@ def test_dtype(simple_dtype):
         np.zeros(1, m.trailing_padding_dtype())
     )
 
+    assert m.test_dtype_kind() == ["b"] + ["i"] * 3 + ["f"] * 3
```

Comment:
If it's up to me, I'd say to just spell it out, as it nicely matches with the list below.

If you really want to make things easy to write, then go for `list("biiifff")` and `list("?hilfdg")`. Just a suggestion. I'm not claiming that's better readable; just claiming it doesn't require opening and closing quotes and adding commas.

Reply:
okay !